_docs/6-baremetal-user.md

```diff
@@ -10,16 +10,32 @@ excerpt: "Welcome to the Bare Metal User page."
 last_modified_at: 2018-01-10T11:22:01-05:00
 ---
 
-# Getting Started
+# Introduction
 
-1. Download the Icicle kit reference design
-2. Follow QSG -> OOBE (Boot linux and load webserver)
-3. Load latest factory images (Linux + U-Boot, HSS, Bitstream
-4. Development Flow (Bare Metal)
-    * Bare Metal FirmWare
-    * Import bare metal SDK
-    * Add / edit bare meatl application
-    * Build Bare metal application as HSS payload
-    * Program bare metal image with USBDMSC
+Before attempting to run a bare metal application, please make sure that you are 
+using the [latest release](https://github.com/polarfire-soc/icicle-kit-reference-design/releases) of the Icicle Kit Reference design. 
```

Comment:
Before attempting to run a bare metal example application on the PolarFire SoC Icicle Kit, make sure you are using the ...